libs/boost/Makefile

```diff
@@ -326,7 +316,6 @@ define DefineBoostLibrary
 
   BOOST_DEPENDS+= +$(if $(4),$(4):boost-$(1),boost-$(1))
   PKG_CONFIG_DEPENDS+= CONFIG_PACKAGE_boost-$(1)
-  HOST_CONFIG_DEPENDS+= CONFIG_boost-host-build-$(1)
```

Comment:
The reason I had it this way was so that we don't unnecessarily compile some of these boost host packages if no host tools will actually use them. This made it so that these host libs get compiled iff there is a host tool that would use it.

Are we fine having these libs compile everytime even if no host tools use it?

Reply:
Sure. It's a host package. It's only needed for compilation. Size is not really a problem for the host.